yt/visualization/_handlers.py

```diff
@@ -374,6 +353,54 @@ def get_norm(self, data: np.ndarray, *args, **kw) -> Normalize:
 
         return norm_type(*args, **kw)
 
+    def _guess_linthresh(self, finite_plot_data):
+        # finite_plot_data is the ImageArray or ColorbarHandler data, already
+        # filtered to be finite values
+
+        # get the extrema for the negative and positive values separately
+        # neg_min -> neg_max -> 0 -> pos_min -> pos_max
+        def get_minmax(data):
+            if len(data) > 0:
+                return np.nanmin(data), np.nanmax(data)
+            return None, None
+
+        pos_min, pos_max = get_minmax(finite_plot_data[finite_plot_data > 0])
+        neg_min, neg_max = get_minmax(finite_plot_data[finite_plot_data < 0])
+
+        has_pos = pos_min is not None
+        has_neg = neg_min is not None
+
+        # the starting guess is the absolute value of the point closest to 0
+        # (remember: neg_max is closer to 0 than neg_min)
+        if has_pos and has_neg:
+            linthresh = np.min((np.abs(neg_max), pos_min))
```

Comment:
Unless you think it's less clear (builtin `abs` would also work btw)
```suggestion
            linthresh = np.min((-neg_max, pos_min))
```

Reply:
I actually think it's a little **more** clear with the `-`! Will change it.